application/controllers/Register.php

```diff
@@ -0,0 +1,134 @@
+<?php
+
+class Register extends FrameWork {
+
+  /**
+   * 
+   * It checks if error is present in the page or not and accordingly  
+   * if it is present then the page is embedded.
+   * 
+   * @return void
+   */
+  
+  public function signup() {
+    $this->view('register');
+    if (isset($_POST['register'])) {
+      if (!(Register::checkName() || Register::checkMail() || Register::checkBirthDay() || Register::checkPass())) {
```

Comment:
```
if (!(Register::checkName() 
  || Register::checkMail() 
  || Register::checkBirthDay() 
  || Register::checkPass())
) {
```

Reply:
In this piece of code, I have done backend validation using the static methods and if anyone's condition returns true then the registration does not continue. Should I make changes to this logic?